test/e2e/common/configmap_volume.go

```diff
@@ -246,7 +246,7 @@ var _ = Describe("[sig-storage] ConfigMap", func() {
 					},
 					{
 						Name:    containerName2,
-						Image:   imageutils.GetE2EImage(imageutils.BusyBox),
+						Image:   busyboxImage,
```

Comment:
Delete busyboxImage var and use imageutils.GetE2EImage(imageutils.BusyBox) everywhere.

Reply:
you means delete "busyboxImage" all package?

everywhere under /test/e2e/common
using "busyBoxImage" and "mountImage" defiend 
/test/e2e/common/util.go

Does this sound to erase all of them?


https://github.com/kubernetes/kubernetes/blob/master/test/e2e/common/util.go#L43-L46

Reply:
Yes, I believe we should be using imageutils instead.

Reply:
ok close this PR.
thanks for review

Reply:
I didn't mean you should close the PR. I meant you should delete the `busyboxImage` variable and change everything to use `imageutils.GetE2EImage(imageutils.BusyBox)`

Reply:
I will upload another PR.  wait sec